vmm/src/memory_manager.rs

```diff
@@ -578,7 +579,7 @@ impl MemoryManager {
                         zone_config.hugepages,
                         zone_config.hugepage_size,
                         zone_config.host_numa_node,
-                        existing_memory_files.remove(&guest_ram_mapping.slot),
+                        existing_memory_files.remove(&guest_ram_mapping.slot.unwrap()),
```

Comment:
Is slot guaranteed to be `Some()` at this point? It would be better to `if let Some(slot)....`